wall_e/src/resources/cogs/reminders.py

```diff
@@ -13,9 +13,14 @@
 import psycopg2
 # import os
 import datetime
+import pytz
 logger = logging.getLogger('wall_e')
 
 
+# Bot's default timezone; if running from another timezone (eg: Canada/Eastern), change accordingly
+wall_e_timezone = pytz.timezone("Canada/Pacific")
```

Comment:
I think this line should really be moved to either the init function or the command itself.

Reply:
My original idea for it was the having it higher up would make it easier to find and edit for anyone hoping to change it for their own instance of the bot, but seeing as it only has one use I think you're right
Fixed it with the newest commit

Reply:
My original idea was that it would be easier to find and edit for someone hosting another instance when defined earlier on, but seeing as how it's only used once I think you're right; I got rid of the `wall_e_timezone` variable entirely
A future idea could be to define a timezone somewhere else more centralized, like in a config file, but that's separate from the functionality of the command itself